cli/tests/unit/test_formatters.py

```diff
@@ -83,11 +83,11 @@ def foo():
         taint_rule_match
     )
 
-    assert bool(thread_flow_location.get("location")), (
+    assert bool(thread_flow_location[0].get("location")), (
```

Comment:
```suggestion
    assert bool(thread_flow_locations[0].get("location")), (
```

Please also rename it `thread_flow_locations` everywhere it's relevant

Reply:
Done